ci-operator/step-registry/configure-registry-storage/set-nutanix-object/configure-registry-storage-set-nutanix-object-commands.sh

```diff
@@ -33,14 +33,17 @@ else
   exit 1
 fi
 
-NUTANIX_OS_ENDPOINT="${CLUSTER_PROFILE_DIR}/nutnaix_os_endpoint"
```

Comment:
Does that mean before we had `nutnaix_os_endpoint` in `nutanix-qe` cluster profile, now it get removed?

Reply:
This is the script issue, if use the command, cannot get nutnaix_os_endpoint environment, so i delete it.

Reply:
I found the initial [PR](https://github.com/openshift/release/pull/39951) which is introduced for the first time, and its debug [log](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/39951/rehearse-39951-periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-nutanix-ipi-registry-nutanix-object-p2-f28/1666753516569366528/build-log.txt) show `configure-registry-storage-set-nutanix-object` succeeded.

So that means "${CLUSTER_PROFILE_DIR}/nutnaix_os_endpoint" existed before, but recently it get changed. I also check that in `nutanix-qe` cluster profile, `nutnaix_os_endpoint`  is shown in `${CLUSTER_PROFILE_DIR}/secrets.sh` for now, but I think someone moved the vault key recently, that introduce a regression script bug, I wonder why we changed the  `nutnaix_os_endpoint` vault key.

Reply:
```
NUTANIX_OS_ENDPOINT="${CLUSTER_PROFILE_DIR}/nutnaix_os_endpoint"
if [[ ${NUTANIX_OS_ENDPOINT} == "" ]]; then
  echo "Did not find nutanix os endpoint, exit now"
  exit 1
fi
```
This is the original code, indeed there were some script issue on that time, but I can not figure why it passed the pj-rehearse job on that time.

Reply:
cc @sgaoshang

Reply:
OK, I'll create file nutnaix_os_endpoint in vault with content 
`https://os.lts-cluster.internal.nutanix-dev.devcluster.openshift.com/`

Reply:
Also removed `nutnaix_os_endpoint='os.lts-cluster.internal.nutanix-dev.devcluster.openshift.com'` from secrets.sh

Reply:
thanks @sgaoshang  create the file, @jianlinliu  updated according your comments, let me check the step result